### DISCUSSION TITLE
Lucia auth lib

### DISCUSSION THREAD

**Discussion by jm61** [pocketbase/pocketbase#641] [Q&A]
Hello to all,
is the [lucia-sveltekit](https://github.com/pilcrowOnPaper/lucia-sveltekit) auth lib can be a complement of PocketBase or its auth mechanism is robust enough?

**Comment by ganigeorgiev** [comment]
It depends on your use case. You can use PocketBase as its database and create an adapter similar to the available in https://github.com/pilcrowOnPaper/lucia-sveltekit/tree/main/packages.

This may be useful if you want to abstract the authentication and to be able to switch between different db/backend providers, but if your app relies heavily on only PocketBase, then you may be better with just the [JavaSript SDK](https://github.com/pocketbase/js-sdk).

**Reply by jm61** [reply]
okay, my thoughts too, I focus on PB and Js sdk, well enough for me! Thanks for your quick reply.
